### PR TITLE
Log contributions per entity during aggregation

### PR DESCRIPTION
**Pull request by noah-aviel-dove**

### CODE REVIEW

File: src/azul/indexer.py [hunk 1]
```diff
@@ -325,6 +325,10 @@ def _aggregate(self, contributions: List[Contribution]) -> List[Aggregate]:
         # Aggregate contributions for the same entity
         aggregates = []
         for entity, contributions in contributions_by_entity.items():
+            log.info('Found %i contributions for entity %s/%s during aggregation',
+                     len(contributions),
```
Comment (hannes-ucsc) [inline src/azul/indexer.py]: These go on one line:

image

File: src/azul/indexer.py [hunk 2]
```diff
@@ -325,6 +325,10 @@ def _aggregate(self, contributions: List[Contribution]) -> List[Aggregate]:
         # Aggregate contributions for the same entity
         aggregates = []
         for entity, contributions in contributions_by_entity.items():
+            log.info('Found %i contributions for entity %s/%s during aggregation',
```
Comment (hannes-ucsc) [inline src/azul/indexer.py]: This is too late for what I was envisioning but may be useful, so leave it in. I had envisioned to log around [here](https://github.com/DataBiosphere/azul/pull/1579/files#diff-c98e1dc4399542337daecdeafc7da4feL297).

```suggestion
            log.info('Selected %i contributions to be aggregated for entity %s/%s.',
```

File: src/azul/indexer.py [hunk 3]
```diff
@@ -294,7 +294,11 @@ def _read_contributions(self, tallies: Tallies) -> List[Contribution]:
             log.info('Reading %i expected contribution(s) using scan().', num_contributions)
             hits = scan(es_client, index=index, query=query, size=page_size, doc_type=Document.type)
         contributions = [Contribution.from_index(self.field_types(), hit) for hit in hits]
-        log.info('Read %i contribution(s).', len(contributions))
+        log.info('Read %i contribution(s):', len(contributions))
+        entity_key = attrgetter('entity')
```
[the remark this answers is not in the document]
Comment (hannes-ucsc) [reply src/azul/indexer.py]: Never mind, it probably doesn't work.

[the remark this answers is not in the document]
Comment (noah-aviel-dove) [reply src/azul/indexer.py]: AttributeError: type object 'Document' has no attribute 'entity'

**Comment by hannes-ucsc** [comment]
The CI failure can be explained to a temporary problem on the Gitlab instance. The Docker networking on that instances seems to have been shot. Rebooting the instance fixed the issue but we don't know what caused the Docker networking to break.

**Pull request merged**